### PR TITLE
ENG-3979 add confirmation when removing references of widget config

### PR DESCRIPTION
**Pull request by jeffgo10**

### CODE REVIEW

File: src/commands/mfe/rm.ts [hunk 1]
```diff
@@ -22,6 +25,48 @@ export default class Rm extends Command {
...
       new MicroFrontendService()
+    const mfeReferences = microFrontendService.findWidgetConfigReferences(
+      args.name
+    )
+
+    if (mfeReferences.length > 0) {
+      const { removeConfigs } = await inquirer.prompt([
+        {
+          name: 'removeConfigs',
+          message: `Reference to ${args.name} will be removed from this bundle's microfrontend widgets. Do you wish to continue?`,
```
Comment (zonia3000) [inline src/commands/mfe/rm.ts]: Could you please replace "this bundle's microfrontend widgets" with the actual list of widgets names? We agreed that when we discussed the AC.

File: src/services/microfrontend-service.ts [hunk 2]
```diff
@@ -90,6 +90,33 @@ export class MicroFrontendService {
     return microfrontends.find(({ name }) => name === mfeName)
   }
 
+  private isConfigMfeSiblingWidget(mfe: MicroFrontend): boolean {
+    return (
+      'configMfe' in mfe &&
+      mfe.configMfe !== undefined &&
+      mfe.configMfe.slice(0, 9) !== 'internal:'
```
Comment (zonia3000) [inline src/services/microfrontend-service.ts]: Maybe we can extract `internal:` to a constant, shared also with `component-service.ts`.

File: src/commands/mfe/rm.ts (hunk 1 above)
Comment (jeffgo10) [reply src/commands/mfe/rm.ts]: done

File: src/services/microfrontend-service.ts (hunk 2 above)
Comment (jeffgo10) [reply src/services/microfrontend-service.ts]: done

**Review by zonia3000 — APPROVED** [review]

**Pull request merged**